emulator/contracts_test.go

```diff
@@ -17,10 +15,18 @@ func TestCommonContractsDeployment(t *testing.T) {
 
 	//only test monotonic and emulator ( mainnet / testnet is used for remote debugging )
 	chains := []flowgo.Chain{
-		flowgo.MonotonicEmulator.Chain(),
 		flowgo.Emulator.Chain(),
+		flowgo.MonotonicEmulator.Chain(),
 	}
 
+	contracts := []string{
```

Comment:
I like this version of the test much better :100: 
Could we also construct the slice from the global variable? So that it automatically pickups additions to it, like here https://github.com/onflow/flow-emulator/pull/428/files#diff-c2ef94ae8feb5831cffce8f3dccf77016473ab207cfbd3ae3632dc2bc99bd5eaR35-R40